config/initializers/assets.rb

```diff
@@ -8,7 +8,7 @@
 # Rails.application.config.assets.paths << Emoji.images_path
 # Add Yarn node_modules folder to the asset load path.
 Rails.application.config.assets.paths << Rails.root.join('node_modules')
-
+Rails.application.config.assets.precompile += %w( cocoon.js )
```

Comment:
Are you sure that this is required? According to documentation you just need https://github.com/nathanvda/cocoon#rails-31rails-4rails-5